src/scss/styles.scss

```diff
@@ -0,0 +1,51 @@
+@import "bootstrap/scss/bootstrap";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style-type: none;
+}
+
+header {
+  margin-top: 2%;
+  margin-left: 1%;
+  font-size: larger;
+}
+
+.title-cont {
+  display: flex;
+}
+
+.main {
+  display: flex;
+}
+
+.submit_btn {
+  display: flex;
+  justify-content: end;
+  margin-left: 10.5%;
+}
+
+.list {
+  border: 4px solid black;
+  border-width: medium;
+  width: 100%;
+  padding: 0;
+}
+
+li {
+  background: white;
+  width: 100%;
+}
+
+li:nth-child(even) {
+  background: lightgray;
+}
```

Comment:
| Your Design | Microverse Wireframe |
|-|-|
| ![image](https://github.com/AnsarIbrahim/LeaderBoard-new/assets/93039359/215e2182-9f1c-47fa-962f-4e04af277332) | ![image](https://github.com/AnsarIbrahim/LeaderBoard-new/assets/93039359/a809833d-776f-415c-af8a-85a6d0f0875f) |

- [ ] It would be nice if you make your contents in the center of the webpage to follow the wireframe so your app will look professional and attractive 👍